packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -3,7 +3,10 @@
 // found in the LICENSE file.
 import 'dart:async';
 import 'dart:convert';
+import 'dart:io';
```

Comment:
do we have to include dart:io here? That will make it hard to use in the web version of the app. I would place the dart:io import behind a conditional import if you need it.

Reply:
Okay replaced dart:io with package:file it supports LocalFileSystem as well as MemoryFileSystem based on kIsWeb value.